spanner_orm/field.py

```diff
@@ -179,9 +179,20 @@ def validate_type(self, value: Any) -> None:
 class String(FieldType):
   """Represents a string type."""
 
+  def __init__(self, size: Optional[int] = None):
```

Comment:
(nit) https://cloud.google.com/spanner/docs/reference/standard-sql/data-definition-language#scalars calls it `length`, so let's use that for consistency. (Same comment below for BytesBase64.)

Reply:
Done.